tests/reconfiguration.py

```diff
@@ -380,7 +380,16 @@ def test_node_filter(network, args):
         def get_nodes(status):
             r = c.get(f"/node/network/nodes?status={status}")
             nodes = r.body.json()["nodes"]
-            return sorted(nodes, key=lambda node: node["node_id"])
+            return sorted(
+                [
+                    {
+                        "node_id": node["node_id"],
```

Comment:
I think this is sufficient fields to include but if anyone knows better, please let me know

Reply:
I think we should keep all the metadata apart from `primary` in this object - we don't expect anything else to change. We could do that as a dict comprehension, but probably simpler to just explicitly delete this field above